package.json

```diff
@@ -7,6 +7,7 @@
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "^5.2.5",
     "@mui/material": "^5.2.7",
+    "cloudinary-react": "^1.7.1",
```

Comment:
Are we using Cloudinary for the platform in prod? 
Just making sure we've vetted their server before committing to using it

cc: @AndrewWestberg